linux/Jenkinsfile

```diff
@@ -575,7 +575,8 @@ def uploadRpmArtifacts(String DISTRO, String rpmArch, String Version) {
     def Distro = "RPM"
     def distro_Package = [
         'redhat' : [
-            'rpm/centos/7',
+            'rpm/centos/9',
+            'rpm/centos/10',
```

Comment:
Thanx a lot for this! I had it in mind for long time! Highly apprecied!

Should not the centos-7 be persisted? AFAIK it it still valid system where jdk should build and run. Maybe not all jdks, but 8-17 for sure.

Reply:
Yeah, I am unsure on the project policy for when to remove EOL distros, but both CentOS 7 and 8 are long gone EOL. The repo has precedent for removing EOL distros (mainly with Fedora I think)

Reply:
Yes, we'd like to leave centos7 for the moment, as its our build platform.